### PR TITLE
[CI] Mention resource type in adapters e2e docs

### PR DESCRIPTION
**Pull request by gr455**
**Description**
Mention the resource type in the auto generated docs for adapters e2e test results in the format `type/resource: "Status"`
This is to avoid situations where two resources have the same name but different types

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by gr455** [comment]
@Revolyssup Can you check this please

File: .github/workflows/test_adapters.yaml [hunk 1]
```diff
@@ -180,7 +180,7 @@ jobs:
                   resourceinfo=$(kubectl get ${types[i]} -n ${ns[i]} | grep ${resources[i]} | tr -s ' ')
                   resourcestarted=$(echo $resourceinfo | cut -d " " -f 1)
                   resourcestatus=$(echo $resourceinfo | cut -d " " -f 3)
-                  if [[ ("${types[i]}" = "pod" && "$resourcestatus" = "Running") || ("${types[i]}" != "pod" && "$resourcestarted" != "") ]];then
+                  if [[ ("${types[i]}" = "pod" && "$resourcestatus" != "Running") || ("${types[i]}" != "pod" && "$resourcestarted" = "") ]];then
```
Comment (gr455) [inline .github/workflows/test_adapters.yaml:183]: A little oversight on my side in the last PR

Comment (Revolyssup) [reply .github/workflows/test_adapters.yaml:183]: Why are you breaking out when pod is not running?

Comment (Revolyssup) [reply .github/workflows/test_adapters.yaml:183]: Oh! gotcha

**Review by Revolyssup — APPROVED** [review]

**Comment by Revolyssup** [comment]
@gr455  Ready for merge?

**Comment by gr455** [comment]
> @gr455 Ready for merge?

Yup!

**Pull request merged**